### PR TITLE
table of contents notebook added

### PR DESCRIPTION
**Pull request by gdicecco**
Creates a new notebook file ('guide_to_notebooks') with some introductory text and links to each notebook

### CODE REVIEW

File: guide_to_notebooks.ipynb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    "\n",
+    "[threshold_tools_exceedance.ipynb](threshold_tools_exceedance.ipynb): Perform calculations and explore visualizations of threshold exceedance events using an interactive graphical user interface (GUI). An extension of the topics introduced in threshold_tools_basics. \n",
+    "\n",
+    "[threshold_tools_application_example.ipynb](threshold_tools_application_example.ipynb): A walk through of two applications of the AE threshold tools for the energy sector: (1) updating design standards and (2) asset-by-asset vulnerability assessments\n",
+    "\n",
+    "### Other Science Applications\n",
+    "[explore_amy.ipynb](explore_amy.ipynb): Explore the concept and applications of an Average Meteorological Year to represent the mean climatological conditions over one year of hourly data.\n",
+    "\n",
+    "explore_warming.ipynb: Explore the concept of Global Warming Levels, which can be used to compare possible climate outcomes across multiple scenarios or model simulations.\n",
```
Comment (elehmer) [inline guide_to_notebooks.ipynb:51]: The last 2 notebooks don't have links to them...

**Review by elehmer — CHANGES_REQUESTED** [review]
Looks and works great but I noticed the last 2 notebooks do not have links...

**Review by elehmer — APPROVED** [review]

File: guide_to_notebooks.ipynb [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1a0d94d-d313-42cf-b45a-0dd31953daf8",
+   "metadata": {},
+   "source": [
+    "Welcome to the Cal-Adapt: Analytics Engine JupyterHub! Here, you can access tools and guidance to explore and analyze climate projections data downscaled for California. \n",
+    "\n",
+    "If you are new to working with Jupyter notebooks, check out our [video tutorial](https://www.youtube.com/watch?v=eEo9V_OJw9w) or review Jupyter's [introductory documentation](https://jupyter.org/try-jupyter/retro/notebooks/?path=notebooks/Intro.ipynb). You can also review our [technical documentation](https://climakitae.readthedocs.io/en/latest/) to learn more about the open source python library that powers these notebooks, `climakitae`.\n",
```
Comment (nicolejkeeney) [inline guide_to_notebooks.ipynb:18]: This is so nitpicky-- sorry in advance-- but the Jupyter documentation capitalizes the "Notebook" in "Jupyter Notebook", so would be good to follow those conventions here as well.

**Review by nicolejkeeney — CHANGES_REQUESTED** [review]
One small change requested regarding capitalization (see comment below) but otherwise looks fantastic. I might rename this notebook something like "AE_navigation_guide" instead of "guide_to_notebooks", to avoid confusion with the "getting_started" notebook, but that's just a personal opinion that you're free to ignore.

**Review by vicford — APPROVED** [review]
Looks good to me as well

File: guide_to_notebooks.ipynb (hunk 2 above)
Comment (gdicecco) [reply guide_to_notebooks.ipynb]: No problem, easy fix!

**Review by nicolejkeeney — APPROVED** [review]
looks great!

**Pull request merged**